po/fr.po

```diff
@@ -640,13 +638,13 @@ msgstr "1. Décompresser %s"
 msgid "2. Copy the contents into %s."
 msgstr "2. Copier le dossier obtenu dans %s."
 
-#, fuzzy, tcl-format
+#, tcl-format
 msgid "[deken] uninstalling '%s'"
-msgstr "[deken]: connecté ? %s"
+msgstr "[deken] désinstallation de %s"
```

Comment:
In regards with the original English string in the msgid, %s should be '%s'

Reply:
of course you're right... thanks

Reply:
:+1: